notebooks/source/prodlda.ipynb

```diff
@@ -0,0 +1,794 @@
+    "\n",
+    "$$\n",
+    "    P = \\{0.12, 0.06, 0.26, 0.12, 0.32, 0.12\\}\n",
+    "$$\n",
+    "\n",
+    "In the general case, however, the probabilities that generated the dataset are hidden to us: they are _latent random variables_. We don't have access to them, only the observations $\\mathcal {D}$. The purpose of probabilistic modeling is to learn the hidden variables from the observations only.\n",
+    "\n",
+    "Suppose we observe N die rolls, $\\mathcal{D} = \\{x_1, ..., x_n\\}$, where $x_i \\in \\{1, ..., 6\\}$. If we assume the data is independent and identically distributed (iid), the likelihood of observing this specific dataset has the form \n",
+    "\n",
+    "$$\n",
+    "    p(\\mathcal{D} | \\theta) = \\prod_{i}^{6} \\theta_{k}^{N_k} \\label{eq:multinomial} \\tag{1}\n",
+    "$$\n",
+    "\n",
+    "\n",
+    "where $N_k$ is the number of times we observed every digit $k$. The likelihood for the **multinomial distribution** has the same form: this is the best distribution to model this simple example. But how to model the latent random variable $\\theta$?\n",
+    "\n",
+    "$\\theta$ is a vector of dimension 6 (one for each digit) that lives in the 6-dimensional probability simplex, i.e. the probabilities are all positive real numbers that add up to 1. There is a natural distribution for such probability vectors: the **Dirichlet distribution**. Indeed the Dirichlet distribution is commonly used as a prior distribution in Bayesian statistics, since it is \"conjugate\" to the multinomial (and categorical) distributions.\n",
+    "\n",
+    "The Dirichlet distribution is parameterized by the vector $\\alpha$, which has the same number of elements as our multinomial parameter $\\theta$ (6 in our example). Although in this simple case we could compute the posterior over the hidden variables $\\theta$ and $\\alpha$ analytically, let's instead see how we can use NumPyro and Markov Chain Monte Carlo (MCMC) to do inference:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import jax.numpy as jnp\n",
+    "import numpyro\n",
+    "import numpyro.distributions as dist\n",
+    "from jax import device_put, lax, random\n",
+    "from numpyro.infer import MCMC, NUTS\n",
+    "from tqdm import trange\n",
+    "\n",
+    "rng_key = random.PRNGKey(0)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "nbsphinx": "hidden",
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "\n",
+    "assert numpyro.__version__.startswith(\"0.6.0\")\n",
+    "# Enable smoke test - run the notebook cells on CI.\n",
+    "smoke_test = \"CI\" in os.environ"
+   ]
+  },
```

Comment:
Adding

```
{
    "nbsphinx": "hidden",
    "tags": []
}
```

to the cell metadata allows you to suppress cells from the rendered output. I think this could be a nice pattern for removing smoke test configuration from the docs, but still allowing notebooks to be run in CI. What do you think?